rmc/utils/missense_badness.py

```diff
@@ -155,8 +158,12 @@ def prepare_amino_acid_ht(
     :param bool include_rescue: Whether to include regional missense OE from rescue RMC results.
         If True, RMC results are derived from the initial and rescue RMC search.
         If False, RMC results are derived from the initial RMC search only.
-    :param bool overwrite: Whether to overwrite temporary data if it already exists.
-        If False, will read existing temporary data rather than overwriting.
+    :param bool overwrite_temp: Whether to overwrite intermediate temporary (OE-independent) data if it already exists.
+        If False, will read existing intermediate temporary data rather than overwriting.
+    :param bool overwrite_output: Whether to entirely overwrite final output (OE-dependent) data if it already exists.
+        If False, will read and modify existing output data rather than overwriting entirely.
```

Comment:
it would add more words, but maybe a note that the modifying just includes adding additional columns would be helpful